app/models/Progress.scala

```diff
@@ -112,6 +122,10 @@ object Progress {
       occupationQuestionnaire = progressResponse.questionnaire.contains("occupation_questionnaire"),
       submitted = progressResponse.submitted,
       withdrawn = progressResponse.withdrawn,
+      eligibleForJobOffer = EligibleForJobOfferProgress(
```

Comment:
I am thinking but not very sure, if this should be jobOffer instead of eligibleForJobOffer? And then jobOffer will contain eligible and eligibleNotified. What do you think?

Reply:
Slightly bigger change. Yeah. I can do it.